clients/flash/shared/ColorPalette.as

```diff
@@ -11,3 +11,19 @@ private const bbbDarkGrey:uint = 0x353B42;
 private const bbbGrey:uint = 0x8B9AA8;
 
 private const bbbWhite:uint = 0xF3F6F9;
+
+// Color names are guessed using http://chir.ag/projects/name-that-color/
+
+private const regentGrey:uint = 0x8A9AA7;

```

Comment:
Similar idea to "greyLight", "greyMedium", "greyDark", but with more granularity for additional shades.